code/src/components/Slider.css

```diff
@@ -0,0 +1,36 @@
+.input-slider {
+  width: 100%;
+  cursor: pointer;
+}
+#rating-steps {
```

Comment:
We should avoid styling by id's - sticking to classes is far more safe. More on that aspect : https://www.drinchev.com/blog/css-with-only-class-names/